geemusic/intents/playback.py

```diff
@@ -81,6 +82,25 @@ def prev_song():
 
         return audio().play(stream_url)
 
+@ask.intent("AMAZON.ShuffleOnIntent")
+def shuffle_tracks():
+    shuffled_songs = queue.song_ids
+    
+    if shuffled_songs == [] or None:
+        return statement("There are no songs to shuffle.")
+    else:
+        api = GMusicWrapper.generate_api()
+    
+        # This will randomly reorder the song ids thereby "shuffling" the tracks
+        random.shuffle(shuffled_songs)
+    
+        # Start streaming the first track in the new shuffled list
+        first_song_id = queue.reset(shuffled_songs)
```

Comment:
does this work? `queue.reset` expects the first argument to be a list of track dicts, not a list of song ids. it might be cleaner to implement this shuffling operation in the `MusicQueue` class rather than in an intent.

If you could implement it in the queue class you could also find a way to cleaning store the original ordering, that way we could also implement an intent for disabling shuffle.